app/models/command.rb

```diff
@@ -27,4 +27,12 @@ def usages
   def self.usage_ids
     MacroCommand.pluck(:command_id) + StageCommand.pluck(:command_id)
   end
+
+  def to_auditable_json
+    to_json(
+      only: [:id, :command, :project_id],
+      include: [
+        stages: { only: [:id] }
+      ])
+  end
```

Comment:
Please use serializers for this, rails' built in as_json/to_json are poor presenters.